float64 marker_x
float64 marker_y
float64 marker_z
---
float64 obj_x
float64 obj_y
float64 obj_z



